datafusion/sqllogictest/test_files/parquet.slt

```diff
@@ -251,27 +251,25 @@ SELECT COUNT(*) FROM timestamp_with_tz;
 ----
 131072
 
-# FIXME(#TODO) fails with feature `force_hash_collisions`
-# https://github.com/apache/datafusion/issues/11660
 # Perform the query:
```

Comment:
```suggestion
```

Reply:
Would be great to add some comments why test is so specific

Reply:
I agree better comments would be nice -- however I don't think this PR makes the comment any more/less cryptic, so I don't think we should hold it for better comments.

Maybe we can improve the comments as a future PR